package.json

```diff
@@ -12,7 +12,7 @@
   "license": "MIT",
   "dependencies": {
     "next": "^12.1.0",
-    "nextra": "^1.1.0",
```

Comment:
We should probably keep the nextra version more up to date, try running `npm remove nextra` and then `npm install nextra`

Reply:
Updated nextra version to 1.1.0. Don't know what happened with that